openhsi/data.py

```diff
@@ -277,10 +275,10 @@ def rad2ref_6SV(self:CameraProperties, x:Array['λ,x',np.float32]) -> Array['λ,
 @patch
 def set_processing_lvl(self:CameraProperties, lvl:int = 2, custom_tfms:List[Callable[[np.ndarray],np.ndarray]] = None):
```

Comment:
just noticed the doc string says -1 is default, but 2 is default in func def.

Reply:
nice catch. I got it in the docs, the tutorial, the docstring, in the __init__ but not the func def itself! 
fixed in upcoming commit.